docs/_docs/Develop/script-js.md

```diff
@@ -200,6 +200,13 @@ var Installer = Java.extend(org.phoenicis.scripts.Installer, installerImplementa
 ### Advanced
 This section describes some advanced methods which give you more possibilities to control your script.
 
+### Include mechanism
+When you want to use a certain functionality in your scripts, you need to include it in your scripts, for example:
+```javascript
+include(["engines", "wine", "quick_script", "steam_script"]);
+```
+allows you to execute a steam script. The content of the include can be determined from the path made up of the id's on the different folder levels leading to the functionality.
```

Comment:
At this point we should give the user an example how the `path` is determined.
When I read `ids on the different folder levels` without knowing about Phoenicis I would not know where to find the mentioned ids.

Reply:
@madoar Hope it is good now.

Reply:
I think it should be `ids` instead of `id's`.
Still I think a short folder example with the relevant example json files would be useful.

Reply:
Or IDs?

Reply:
This is easier to explain since the new include mechanism.